queries/python.scm

```diff
@@ -344,14 +343,6 @@
   (#not-parent-type? @_.removal case_clause)
 ) @_.domain
 
-;;!! for name in value:
```

Comment:
The scope fixtures showed me that we had two different definitions for this same pattern. Really shows why we need them :)